eva/udfs/decorators/yolo_object_detection_decorators.py

```diff
@@ -0,0 +1,207 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+
+import pandas as pd
+
+from eva.catalog.catalog_type import NdArrayType
+from eva.models.catalog.frame_info import FrameInfo
+from eva.models.catalog.properties import ColorSpace
+from eva.udfs.abstract.pytorch_abstract_udf import PytorchAbstractClassifierUDF
+from eva.udfs.decorators.decorators import forward, setup
+from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe, PyTorchTensor
+
+try:
+    import torch
+    from torch import Tensor
+
+except ImportError as e:
+    raise ImportError(
+        f"Failed to import with error {e}, \
+        please try `pip install torch`"
+    )
+
+
+class YoloDecorators(PytorchAbstractClassifierUDF):
+    """
+    Arguments:
+        threshold (float): Threshold for classifier confidence score
+
+    """
+
+    @property
+    def name(self) -> str:
+        return "yolo"
+
+    @setup(cachable=True, udf_type="object_detection", batchable=True)
+    def setup(self, threshold=0.85):
+        self.threshold = threshold
+        self.model = torch.hub.load("ultralytics/yolov5", "yolov5s", verbose=False)
+
+    @property
+    def input_format(self) -> FrameInfo:
```

Comment:
We don't need this anymore, right?

Reply:
If it fails because it is an abstract method, you can get rid of it from the base class itself.